test/helpers.bash

```diff
@@ -8,6 +8,17 @@ if [ "$(id -u)" != "0" ]; then
     exit 1
 fi
 
+function give_user_ownership() {
+   if [ "$PRIVILEGE_LEVEL" = "priv" ]; then
+      return
+   fi
+   if [ -z "$SUDO_UID" ]; then
+      echo "PRIVILEGE_LEVEL=$PRIVILEGE_LEVEL but empty SUDO_USER"
```

Comment:
SUDO_USER or SUDO_UID ? both are present, but your error prints a different ENV value than what is checked.
```suggestion
      echo "PRIVILEGE_LEVEL=$PRIVILEGE_LEVEL but empty SUDO_UID"
```